### DISCUSSION TITLE
"Spikes" of stars at repositories' creation dates

### DISCUSSION THREAD

**Discussion by akuru-q** [emanuelef/daily-stars-explorer#146] [General]
Hi, a lot of repos start out with a ton of stars right on their creation date, which seems kinda odd.
The worst offender we found so far is the [Bootstrap repo](https://emanuelef.github.io/daily-stars-explorer/#/twbs/bootstrap), which has 23k stars on its supposed creation date.

Does anyone have an explanation or theory for this?

**Comment by emanuelef** [comment]
@akuru-q Have you tried asking GitHub directly ? Or maybe on Reddit https://www.reddit.com/r/github/.

**Comment by emanuelef** [comment]
I temporarily implemented a check on the number of stars received on the first day of the repository. If the number of stars is higher than or equal to the 98th percentile, the first day's data is excluded.
This is only happening when rendering the data, the API call will return all the history including the first day and the same when exporting to CSV or JSON.
